### PR TITLE
Sync security.adoc with quarkusio/quarkus/blob/main/SECURITY.md

### PR DESCRIPTION
**Pull request by Eng-Fouad**
** If you are updating a guide, please submit your pull request to the main repository: https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc **

### CODE REVIEW

File: security.adoc [hunk 1]
```diff
@@ -50,9 +50,15 @@ The community will fix security bugs for the latest major.minor version publishe
 |Latest 3.x
 |✅
 
+|3.15 LTS
+|✅
+
 |3.8 LTS
 |✅
 
+|3.2 LTS
```
Comment (maxandersen) [inline security.adoc:59]: no, 3.2 LTS is not updated for security and bugs so should not be added.

Comment (gastaldi) [reply security.adoc:59]: We need to remove that from https://github.com/quarkusio/quarkus/blob/main/SECURITY.md

Comment (Eng-Fouad) [reply security.adoc]: > no, 3.2 LTS is not updated for security and bugs so should not be added.

Removed 3.2 LTS :)

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Thanks!

**Pull request merged**